### PR TITLE
fix deploy packages and set dev/stage to RAILS_ENV=production

### PR DESCRIPTION
**Pull request by drh-stanford**
@mejackreed this cleans up the deploys a bit and puts stage into production mode

### CODE REVIEW

File: config/deploy/dev.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-set :bundle_without, %w[sqlite production test].join(' ')
+set :bundle_without, %w[production].join(' ')
```
Comment (mejackreed) [inline config/deploy/dev.rb]: Does this mean `pg` is not available on dev?

Comment (drh-stanford) [reply config/deploy/dev.rb]: yes. we should probably switch -dev to use RAILS_ENV=production too. I only use dev to test the index. what are you using -dev for, if anything?

Comment (mejackreed) [reply config/deploy/dev.rb]: Just to randomly test application code. I'm 👍 for the change.

**Comment by drh-stanford** [comment]
ok, i've switched both dev and stage to use RAILS_ENV=production

**Pull request merged**